### ISSUE TITLE
answer given as challenge

### ISSUE THREAD

**Issue by mwangsmo** [FreeCodeCamp/FreeCodeCamp#5894]
Challenge [Waypoint: Assignment with Minus Equals](http://www.freecodecamp.com/challenges/waypoint-assignment-with-minus-equals#?solution=var%20a%20%3D%2011%3B%0Avar%20b%20%3D%209%3B%0Avar%20c%20%3D%203%3B%0A%0A%2F%2F%20Only%20modify%20code%20below%20this%20line%0A%0Aa%20-%3D%206%3B%0Ab%20-%3D%2015%3B%0Ac%20-%3D%201%3B%0A%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64; rv:43.0) Gecko/20100101 Firefox/43.0</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript
var a = 11;
var b = 9;
var c = 3;

// Only modify code below this line

a -= 6;
b -= 15;
c -= 1;


```

**Comment by raisedadead** [comment]
What exactly is the issue?

**Comment by mwangsmo** [comment]
It isn't a problem, it was my fault.  I had left a browser session open that wasn't refreshed and thought I was started at the beginning of the javascript project but had actually completed a number of the challenges.  It had stored my code sample and when I started working through the challenges, it showed my finished code instead of the challenge.

Once I stepped back to the overview and refreshed the browser, everything cleared up and I was able to jump back to the correct challenge.
